client/sysinfo/src/sysinfo.rs

```diff
@@ -58,9 +121,9 @@ pub(crate) fn benchmark<E>(
 		}
 	}
 
-	let score = ((size * count) as f64 / elapsed.as_secs_f64()) / (1024.0 * 1024.0);
+	let score = Throughput((size * count) as f64 / elapsed.as_secs_f64());
 	log::trace!(
-		"Calculated {} of {:.2}MB/s in {} iterations in {}ms",
```

Comment:
The `B/s` should be unnecessary since the `Display` impl should automatically pick the unit now, no?